src/cljs/rems/application.cljs

```diff
@@ -431,6 +434,23 @@
          {:phase :approve :text :t.phases/approve}
          {:phase :result :text :t.phases/approved}]))
 
+(defn- application-copy-notice [application]
+  (let [old-app (:application/copied-from application)
+        new-apps (:application/copied-to application)]
+    (when (or old-app new-apps)
+      [:<>
+       " ("
+       (when old-app
+         [:<> (text :t.applications/copied-from) " " [application-link old-app nil]])
+       (when (and old-app new-apps)
+         "; ")
+       (when new-apps
+         (into [:<> (text :t.applications/copied-to) " "]
+               (->> (for [new-app new-apps]
```

Comment:
[nit] I'd prefer a direct call instead of threading here. If you want to keep threading, also thread the `into`